vdb-bench-assembly/plugins/vdb-bench-widgets/gitCredentialsControl.js

```diff
@@ -36,8 +43,19 @@
         return directive;
     }
 
-    function GitCredentialsControlController($translate, StorageService, $scope, $base64, $window) {
+    function GitCredentialsControlController($translate, 
+                                             StorageService, 
+                                             $scope, 
+                                             $base64, 
+                                             $window, 
+                                             $rootScope) {
         var vm = this;
+        vm.protocol = {
+            "useHttp": true,
+            "useSsh": false
```

Comment:
What about using neither? This is not a daft question since a git repository with no authentication could be added onto the local filesystem, eg. I run my server locally and also create a git repo on my own machine to get the artifacts out of the workspace and into a version-controlled repo. This is a major possibility and a viable use-case.